### PR TITLE
Fix deprecation warning in units.py

### PR DESCRIPTION
**Pull request by swryan**
Pivotal #161297707

### CODE REVIEW

File: openmdao/utils/units.py [hunk 1]
```diff
@@ -715,7 +718,13 @@ def import_library(libfilepointer):
     _UNIT_CACHE = {}
     _UNIT_LIB = ConfigParser()
     _UNIT_LIB.optionxform = _do_nothing
-    _UNIT_LIB.readfp(libfilepointer)
+
+    # New in Python 3.2: read_file() replaces readfp().
+    if sys.version_info[0] >= 3 and sys.version_info[1] >= 2:
```
Comment (42B) [inline openmdao/utils/units.py]:
```python
if sys.version_info >= (3, 2):  # ?
```

File: openmdao/utils/units.py [hunk 2]
```diff
@@ -769,7 +778,13 @@ def update_library(filename):
     try:
         cfg = ConfigParser()
         cfg.optionxform = _do_nothing
-        cfg.readfp(inp)
+
+        # New in Python 3.2: read_file() replaces readfp().
+        if sys.version_info[0] >= 3 and sys.version_info[1] >= 2:
```
Comment (42B) [inline openmdao/utils/units.py]:
```python
if sys.version_info >= (3, 2):  # ?
```

File: openmdao/utils/units.py (hunk 1 above)
Comment (swryan) [reply openmdao/utils/units.py]: better, thanks!

Comment (42B) [reply openmdao/utils/units.py]: @swryan 

No problem!

**Pull request merged**